monkestation/code/modules/divine_warning/code/effects.dm

```diff
@@ -0,0 +1,79 @@
+#define HEAVEN_TIMETOGO (1<<0)
+#define HEAVEN_TIMETOGO_PLAYED (1<<1)
+
+/atom/movable/screen/fullscreen/divine
+	icon = 'monkestation/code/modules/divine_warning/icons/divine_warning.dmi'
+	icon_state = "he_waits_for_you"
+	layer = EMISSIVE_LAYER_UNBLOCKABLE
+	alpha = 120
+
+/mob/living/proc/flash_divine_overlay(alpha = 120, soundvolume = 80, time = 2 SECONDS)
+	if(client?.prefs)
+		if(client.prefs.read_preference(/datum/preference/toggle/darkened_flash))
+			clear_fullscreen("divine", time)
+			return
+	if (world.time - last_divine_sound > 2.5 SECONDS)
```

Comment:
use the [cooldown macros](https://github.com/Monkestation/Monkestation2.0/blob/master/code/__DEFINES/cooldowns.dm#L91-L107)